src/Stratis.Bitcoin.IntegrationTests/ProofOfStakeSteps.cs

```diff
@@ -57,31 +62,50 @@ public void MineGenesisAndPremineBlocks()
 
         public void MineCoinsToMaturity()
         {
-            this.nodes[this.PremineNode].GenerateStratisWithMiner(100);
+            this.nodes[this.PremineNode].GenerateStratisWithMiner(Convert.ToInt32(this.nodes[this.PremineNode].FullNode.Network.Consensus.CoinbaseMaturity));
             this.sharedSteps.WaitForNodeToSync(this.nodes[this.PremineNode]);
         }
 
         public void PremineNodeMinesTenBlocksMoreEnsuringTheyCanBeStaked()
         {
-            this.nodes[this.PremineNode].GenerateStratisWithMiner(Convert.ToInt32(this.nodes[this.PremineNode].FullNode.Network.Consensus.CoinbaseMaturity));
+            this.nodes[this.PremineNode].GenerateStratisWithMiner(10);
```

Comment:
should that not be "10 + maturity" instead of 10?

Reply:
@bokobza `this.nodes[this.PremineNode].GenerateStratisWithMiner(10);` is correct as we want to advance the node 10 blocks so that they coins become "stake-able"

Reply:
ok so I guess before we mined coins of quantity "maturity" and that was too much?